core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -186,6 +186,25 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
     }
   }
 
+  def getRuntimeOptions(googleProjectId: String, bucketName: String)(implicit executionContext: ExecutionContext): Future[Option[JsValue]] = {
```

Comment:
This seems to be assuming that the only entry in `defaultRuntimeOptions` is `zones`. We probably want to merge with the default options instead of generating a new set.

Reply:
Currently the only entry in `defaultRuntimeOptions` is `zones` based on [rawls.conf in fc-develop](https://github.com/broadinstitute/firecloud-develop/blob/6f37dd2a61ed30bc1fb9ae9bc2707a48e38cfa1c/base-configs/rawls/rawls.conf.ctmpl#L346) and what is being passed to SubmissionActor in rawls [here](https://github.com/broadinstitute/rawls/blob/45a0b0b5b6bd1731bc61736ec71a85a9c2bbfd9d/core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/BootMonitors.scala#L176). So instead of merging it we replaced it

Reply:
I have a feeling that the reference disk work might need/want to use workflow options too

Reply:
I have modified the code to only change the `zones` in the `defaultRuntimeOptions` and not generate a new set

Reply:
I also modified the test to check that if another default runtime option is passed than it should not be overridden by the get-compute-zones logic